cartridges/int_algolia/cartridge/scripts/algolia/lib/algoliaData.js

```diff
@@ -7,11 +7,14 @@ var URLUtils = require('dw/web/URLUtils');
 var Resource = require('dw/web/Resource');
 var logHelper = require('*/cartridge/scripts/algolia/helper/logHelper');
 
+const version = require('*/version').version;
```

Comment:
Instead of doing `require('*/version').version`, we could extend this file to be a more general store for values, configurations in the future that we don't necessarily want to expose as site preferences, and rename it to `algoliaConfig.json`, for example. What do you think?

Reply:
Sure, it doesn't cost anything to do that now, done!